src/main/java/net/fabricmc/loom/build/IntermediaryNamespaces.java

```diff
@@ -37,6 +37,14 @@ public static String intermediary(Project project) {
 		return intermediaryNamespace(project).toString();
 	}
 
+	/**
+	 * Returns the runtime intermediary namespace of the project.
```

Comment:
Nit: maybe `production` instead? There is a dev runtime too.

Reply:
this also applies in dev runtime I believe

Reply:
Isn't dev runtime `named` though